.github/workflows/pkgdown.yaml

```diff
@@ -45,7 +53,37 @@ jobs:
         run: R CMD INSTALL .
 
       - name: Deploy package
+        if: contains(env.isPush, 'true')
         run: |
           git config --local user.email "actions@github.com"
           git config --local user.name "GitHub Actions"
           Rscript -e 'pkgdown::deploy_to_branch(new_process = FALSE)'
+
+      - name: Create website and another page
+        if: contains(env.isPush, 'false')
+        run: |
+          Rscript -e 'pkgdown::build_site()'
+          Rscript -e 'pkgdown::build_home(override=list(destination="cosmo", template = list(params=list( bootswatch = "cosmo"))))'
+
+      - name: Move files
+        if: contains(env.isPush, 'false')
+        run: |
+          cp -r ./cosmo/dev ./docs/dev/cosmo
+
+
+      - name: Deploy to Netlify
+        if: contains(env.isPush, 'false') && contains(env.isExtPR, 'false')
```

Comment:
Where is `env.isExtPR` set?

Reply:
forgot to remove it, it was from before the whole workflow was skipped for external PRs.